.github/workflows/release-drafter.yml

```
@@ -0,0 +1,16 @@
+name: Release Drafter
+
+on:
+  push:
+    # branches to consider in the event; optional, defaults to all
+    branches:
+      - master
+
+jobs:
+  update_release_draft:
+    runs-on: ubuntu-latest
+    steps:
+      # Drafts your next Release notes as Pull Requests are merged into "master"
```

```suggestion
      # Drafts your next Release notes as Pull Requests are merged into "main"
```
I've renamed the default branch.